base/uk.ac.stfc.isis.ibex.ui.scriptgenerator/src/uk/ac/stfc/isis/ibex/ui/scriptgenerator/views/ScriptGeneratorView.java

```diff
@@ -54,287 +63,312 @@
+            topBarComposite.setLayout(new GridLayout(6, false));
+            topBarComposite.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
+
+            // Composite to contain help strings from configs
+            Composite configComposite = new Composite(topBarComposite, SWT.NONE);
+            configComposite.setLayout(new GridLayout(5, false));
+            configComposite.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, true, true, 1, 1));
+
+            // The label for the config selector drop down
+            Label configSelectorLabel = new Label(configComposite, SWT.NONE);
+            configSelectorLabel.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, true, 1, 1));
+            configSelectorLabel.setText("Config:");
+
+            // Drop-down box to select between configs.
+            configSelector = setUpConfigSelector(configComposite);
+
+            // Separate help and selector
+            new Label(configComposite, SWT.SEPARATOR | SWT.VERTICAL);
+
+            // Label for config help
+            Label helpLabel = new Label(configComposite, SWT.NONE);
+            helpLabel.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
+            helpLabel.setText("Config Help: ");
+
+            // Display help for the config
+            Text helpText = new Text(configComposite, SWT.BORDER | SWT.READ_ONLY | SWT.WRAP | SWT.MULTI | SWT.V_SCROLL);
+            var helpTextDataLayout = new GridData(SWT.BEGINNING, SWT.BEGINNING, false, false, 1, 1);
+            helpTextDataLayout.widthHint = 400;
+            helpTextDataLayout.heightHint = 100;
+            helpText.setLayoutData(helpTextDataLayout);
+            helpText.setBackground(clearColor);
+            // Display the correct starting text
+            scriptGeneratorViewModel.getConfig().ifPresentOrElse(
+                config -> {
+                    Optional.ofNullable(config.getHelp()).ifPresentOrElse(
+                            helpString -> helpText.setText(helpString),
+                            () -> helpText.setText("")
+                    );
+                },
+                () -> helpText.setText("")
+            );
+
+            // Composite for the Open Manual button
+            Composite manualButtonComposite = new Composite(topBarComposite, SWT.NONE);
+            manualButtonComposite.setLayout(new GridLayout(1, false));
+            manualButtonComposite.setLayoutData(new GridData(SWT.RIGHT, SWT.CENTER, false, false, 1, 1));
+
+            // Button to open the user manual in a button
+            final Button manualButton = new Button(manualButtonComposite, SWT.NONE);
+            manualButton.setText("Open Manual");
+            manualButton.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 1, 1));
+            
+            try {
+                URL userManualUrl = scriptGeneratorViewModel.getUserManualUrl().get();
```

Comment:
Just as a note for when you are reworking this, a more robust method for this is using the Optional.ifPresentOrElse method https://www.geeksforgeeks.org/optional-ifpresentorelse-method-in-java-with-examples/